src/Models/ReflectionContainer.php

```diff
@@ -50,6 +56,12 @@ public function dataObjectPropertyColumns(
             $this->column_reflection[$class][$property->getName()] = $attribute[0]->newInstance();
         }
 
+        // If the props didn't generate any attributes, set an empty array for this class.
+        if (!array_key_exists($class, $this->column_reflection))
+        {
+            $this->column_reflection[$class] = [];
```

Comment:
Place this single statement before the `foreach`.  Then, the surrounding condition can go away (`if (!array_key_exists($class, $this->column_reflection))`) because it will always be `true`.  And the check for empty props can be deleted, too.